From: "David S. Miller" <davem@davemloft.net>
Date: Thu, 26 Jan 2012 16:25:55 -0500
Subject: e1000e: Need to include vmalloc.h
Patch-mainline: v3.4-rc1
Git-commit: c85c21ad1fe18daf26bfe8684df974d88cec50b7
References: bnc#795174 FATE#313648

Otherwise (on sparc64):

drivers/net/ethernet/intel/e1000e/ethtool.c:657:3: error: implicit declaration of function 'vmalloc' [-Werror=implicit-function-declaration]

Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Benjamin Poirier <bpoirier@suse.de>
---
 drivers/net/ethernet/intel/e1000e/ethtool.c |    1 +
 1 file changed, 1 insertion(+)
--- a/drivers/net/ethernet/intel/e1000e/ethtool.c
+++ b/drivers/net/ethernet/intel/e1000e/ethtool.c
@@ -34,6 +34,7 @@
 #include <linux/pci.h>
 #include <linux/slab.h>
 #include <linux/delay.h>
+#include <linux/vmalloc.h>
 
 #include "e1000.h"
 
